src/udev/udev-builtin-input_id.c

```diff
@@ -244,6 +250,7 @@ static bool test_key(struct udev_device *dev,
                      const unsigned long* bitmask_key,
                      bool test) {
         unsigned i;
+        unsigned j;
```

Comment:
Please don't declare this here, move it into the inner `for i..` loop as `for unsigned j = 0`. Let's not make the scope of these single-letter variables larger than absolutely necessary.